privacyidea/lib/token.py

```diff
@@ -2980,22 +2980,26 @@ def regenerate_enroll_url(serial: str, request: Request, g) -> Union[str, None]:
     return enroll_url
 
 
-def export_tokens(tokens: list[TokenClass]) -> list[dict]:
+def export_tokens(tokens: list[TokenClass], export_user: bool = True) -> list[dict]:
     """
     Takes a list of tokens and returns an dict with all infos.
 
     :param tokens: list of token objects
+    :param export_user: bool
     :return: list of dict with token information
     """
-    exported_tokens = [token.export_token() for token in tokens]
+    exported_tokens = [token.export_token(export_user=export_user) for token in tokens]
```

Comment:
This fails if the export of one of the token fails (unsupported token type).
We should do this in a loop and catch exceptions for each token. We could return a dictionary or a dataclass the information that the export failed for x token.